.github/workflows/verify-swagger-gen.yml

```diff
@@ -0,0 +1,26 @@
+name: Verify
```

Comment:
How about we name this something like `Verify code generation integrity`? 
Sounds fancy, but describes well what happens.